packages/node/src/indexer/blockDispatcher/base-block-dispatcher.ts

```diff
@@ -22,7 +22,7 @@ export interface IBlockDispatcher {
     runtimeService?: RuntimeService,
   ): Promise<void>;
 
-  enqueueBlocks(heights: number[]): void;
+  enqueueBlocks(heights: number[], rawBlocks?: number[]): void;
```

Comment:
I assume the worker block dispatcher needs updating too? Ideally we can reuse the new logic so we dont need to duplicate it in each implmentation